src/test/scala/no/gdl/bookapi/repository/InTranslationRepositoryTest.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Part of GDL book_api.
+ * Copyright (C) 2018 Global Digital Library
+ *
+ * See LICENSE
+ */
+
+package no.gdl.bookapi.repository
+
+import io.digitallibrary.language.model.LanguageTag
+import no.gdl.bookapi.TestData.Domain.DefaultinTranslation
+import no.gdl.bookapi.{IntegrationSuite, TestEnvironment}
+
+class InTranslationRepositoryTest extends IntegrationSuite with TestEnvironment with RepositoryTestHelpers {
+
+  override val inTranslationRepository = new InTranslationRepository
+
+  test("that inTranslation is added and retrieved") {
+    withRollback { implicit session =>
+
+      val added = inTranslationRepository.add(DefaultinTranslation)
+      val retrieved = inTranslationRepository.withId(added.id.get)
+
+      retrieved.head.id should equal(added.id)
+      retrieved.head.userIds should equal(added.userIds)
+      retrieved.head.fromLanguage should equal(added.fromLanguage)
+      retrieved.head.toLanguage should equal(added.toLanguage)
+      retrieved.head.crowdinProjectId should equal(added.crowdinProjectId)
```

Comment:
Do we need to be explicit about the fields, or could these 5 lines be replaced by `retrieved should equal(Some(added))`?

Reply:
No, the equals will report error if any of the values are different.